### PR TITLE
EPMRPP-89221 || status refine contains skipped when clicking on the t…

### PR DESCRIPTION
**Pull request by BlazarQSO**
…otal statistics for widget with excluded skipped items

### CODE REVIEW

**Review by Vadim73i — APPROVED** [review]

File: app/src/components/widgets/multiLevelWidgets/componentHealthCheckTable/columns/statisticsColumn.jsx [hunk 1]
```diff
@@ -48,6 +51,7 @@ export const StatisticsColumn = (
       type: TEST_ITEM_PAGE,
       payload: linkPayload,
     },
+    excludeSkipped,
```
Comment (AmsterGet) [inline app/src/components/widgets/multiLevelWidgets/componentHealthCheckTable/columns/statisticsColumn.jsx:54]: It's redundant, pls, remove.

File: app/src/controllers/testItem/selectors.js [hunk 2]
```diff
@@ -300,47 +300,62 @@ export const statisticsLinkSelector = createSelector(
...
+      'filter.has.levelAttribute': levelAttribute,
       providerType,
       [providerTypeModifierId]: ownProps[providerTypeModifierId],
       launchesLimit,
       isLatest,
-      ...(ownProps.baselineLaunchId && { baselineLaunchId: ownProps.baselineLaunchId }),
-      ...(ownProps.launchId && { launchId: ownProps.launchId }),
+      ...(baselineLaunchId && { baselineLaunchId }),
+      ...(launchId && { launchId }),
+      excludeSkipped,
```
Comment (AmsterGet) [inline app/src/controllers/testItem/selectors.js:344]: I suppose to meet the ticket requirements it is enough to just not provide 'skipped' param while navigating from the widget.
The filters on test item level shouldn't be changed as they are just reflect to the query provided. If it doesn't contain 'skipped', it won't be selected in refines, but should be visible for user to switch it on/off as all other filters.

File: app/src/pages/inside/common/filterEntitiesGroups/stepLevelEntities.jsx [hunk 3]
```diff
@@ -523,7 +523,10 @@ export class StepLevelEntities extends Component {
               label: intl.formatMessage(messages.LaunchStatusInProgress),
               value: IN_PROGRESS.toUpperCase(),
             },
-          ],
+          ].filter(
+            (option) =>
+              !(query.excludeSkipped === 'true' && option.value === SKIPPED.toUpperCase()),
+          ),
```
Comment (AmsterGet) [inline app/src/pages/inside/common/filterEntitiesGroups/stepLevelEntities.jsx:529]: Same here.

File: app/src/pages/inside/common/launchSuiteGrid/executionStatistics/executionStatistics.jsx [hunk 4]
```diff
@@ -42,6 +43,7 @@ export const ExecutionStatistics = ({
         listViewLinkParams={listViewLinkParams}
         className={cx('value', { bold })}
         ownLinkParams={ownLinkParams}
+        excludeSkipped={excludeSkipped}
```
Comment (AmsterGet) [inline app/src/pages/inside/common/launchSuiteGrid/executionStatistics/executionStatistics.jsx:46]: And here.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**